openacademy/models/openacademy_partner.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models
+from odoo import fields
```

Comment:
Just one line to import from the same source
```suggestion
from odoo import fields, models
```